zephyr/CMakeLists.txt

```diff
@@ -74,9 +85,55 @@ if(CONFIG_SOC_ESP32)
     ../components/driver/periph_ctrl.c
     ../components/log/log_noos.c
     ../components/log/log.c
+    adapter/src/common/dport_access.c
     adapter/src/stubs.c
     )
 
+  ## shared WIFI/BT resources
+  if (CONFIG_BT OR CONFIG_WIFI_ESP32)
+    zephyr_sources(
+      ../components/esp_wifi/src/phy_init.c
+      adapter/src/common/esp_system_api.c
+      )
+
+    zephyr_link_libraries(
+      net80211
+      core
```

Comment:
Question: the wifi related libraries are all needed even when using BT only?

Reply:
yes, it is necessary as it shares phy and radio stuff.